src/makeRobotAccountant.js

```diff
@@ -6,7 +6,18 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
-}
+  let count = 4;
+  const argA = (a) => {
+    const argB = (b) => {
```

Comment:
suggestion: give this function different names, `argA` and `argB` are inappropriate here